index.bs

```diff
@@ -4149,7 +4149,7 @@ To <dfn>trigger aggregatable attribution</dfn> given an [=attribution trigger=]
 1. If the result of running [=check if attribution should be blocked by rate limits=]
     with |trigger|, |sourceToAttribute|, and |rateLimitRecord| is not null,
     return it.
-1. If |sourceToAttribute|'s [=attribution source/number of aggregatable attribution reports=] value is equal to [=max aggregatable reports per source=][0], then:
+1. If |sourceToAttribute|'s [=attribution source/number of aggregatable attribution reports=] value is equal to [=max aggregatable reports per source=][0] and |trigger|'s [=attribution trigger/trigger context ID=] is null, then:
```

Comment:
Should we update the name and/or definition of `max aggregatable reports per source` to make it clear that it only applies without trigger context IDs?